composer.json

```diff
@@ -10,7 +10,8 @@
     },
```

Comment:
would import the code without changing its namespaces if a repository merge is desired, and expand the set of configured namespaces here to include `eLife\ApiClient` to provide an easy transition?

Reply:
It's not a merger - more a relocation of a particular set of classes, creating more cohesion and atomic change in a single repo rather than across two. I did have a shim at one point, but because not many projects use these classes directly (a grand total of [5](https://github.com/search?q=org%3Aelifesciences%20%22eLife%5CApiClient%5CApiClient%22%20%20NOT%20repo%3Aelifesciences%2Fapi-sdk-php%20NOT%20repo%3Aelifesciences%2Fapi-client-php&type=code)) Nathan felt it would be easier to just fix up on update rather than more elaborate migration scenario